### DISCUSSION TITLE
Lend some money to others

### DISCUSSION THREAD

**Discussion by azzamsa** [ananthakumaran/paisa#225] [Q&A] [answered]
Hi, 👋

Is there any workflow in Paisa when we lend money to others?

Thank you! 🍦

**Comment by ananthakumaran** [comment, marked as answer]
This has been discussed a couple of times on matrix [chat](https://matrix.to/#/!MGXZKKbIYcgSSNHvuL:matrix.org/$HG-zRoTAgcOEXbPk1KZbFUhozHi6iLSurIQm9HTSYus?via=matrix.org&via=kde.org&via=matrix.cccgoe.de)

**Reply by azzamsa** [reply]
It would be very useful if Paisa forum is searchable and indexable.
